app/views/hyku/api/v1/work/_work.json.jbuilder

```diff
@@ -14,6 +14,15 @@ json.buy_book work.try(:buy_book)
 json.challenged work.try(:challenged)
 json.cname @account.cname
 #                                         "collections" => nil,
+creator = work.creator.try(:first)
+if creator
+  json.creator JSON.parse(creator)
+end
```

Comment:
For these JSON fields, do you think we need to return an empty array in the response JSON if no values exist?
```suggestion
json.creator creator.present? ? JSON.parse(creator) : []
```